### ISSUE TITLE
NextCloud Windows Client 3.2.1 Fails to Sync on New Install

### ISSUE THREAD

**Issue by freeone3000** [nextcloud/desktop#3326]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Expected behaviour
Tell us what should happen

### Actual behaviour
Tell us what happens instead

### Steps to reproduce
1.
2.
3.

### Client configuration
Client version:

Operating system: Windows , Version 10.0.19041 Build 19041

OS language: English

Qt version used by client package (Linux only, see also Settings dialog):
NA

Client package (From Nextcloud or distro) (Linux only):
NA

Installation path of client:
C:\Program Files\Nextcloud

### Server configuration

Nextcloud version:
21.0.1

Storage backend (external storage):
WD Red 6TB in RAID1

### Logs

Please use Gist (https://gist.github.com/) or a similar code paster for longer
logs.

1. Client logfile: 
Since 3.1: Under the "General" settings, you can click on "Create Debug Archive ..." to pick the location of where the desktop client will export the logs and the database to a zip file.
On previous releases: Via the command line: `nextcloud --logdebug --logwindow` or `nextcloud --logdebug --logfile log.txt`
(See also https://docs.nextcloud.com/desktop/3.0/troubleshooting.html#log-files)

[nextcloud_3.2.1_debug_archive.zip](https://github.com/nextcloud/desktop/files/6489794/nextcloud_3.2.1_debug_archive.zip)

2. Web server error log:

3. Server logfile: nextcloud log (data/nextcloud.log):

**Comment by allexzander** [comment]
@freeone3000 

How does the error look for you in the GUI?

From your logs:
`Line 120380: 2021-05-15 23:47:56:310 [ info nextcloud.gui.activity ]:	Updating Activity/Notification/Error view.
	Line 122145: 2021-05-16 09:25:26:222 [ warning nextcloud.sync.networkjob ]:	QNetworkReply::OperationCanceledError "Connection timed out" QVariant(Invalid)
	Line 122146: 2021-05-16 09:25:26:222 [ warning nextcloud.sync.credentials.webflow ]:	QNetworkReply::OperationCanceledError
	Line 122148: 2021-05-16 09:25:26:222 [ info nextcloud.sync.networkjob.jsonapi ]:	JsonApiJob of QUrl("https://10.0.0.30:5569/ocs/v2.php/apps/notifications/api/v2/notifications?format=json") FINISHED WITH STATUS "OperationCanceledError Connection timed out"
	Line 122149: 2021-05-16 09:25:26:222 [ warning nextcloud.sync.networkjob.jsonapi ]:	Network error:  "ocs/v2.php/apps/notifications/api/v2/notifications" "Connection timed out" QVariant(Invalid)
	Line 124599: 2021-05-16 12:32:16:987 [ info nextcloud.gui.folder ]:	Saved folder "1" to settings, status QSettings::NoError`

I can see `OperationCanceledError ` like the server is unreachable or the net speed is slow. Is the server available when you ping it? Does version 3.1.3 works fine but 3.2.1 fails?

**Comment by freeone3000** [comment]
Server is available continually, it's on the local network. Net speed is 1 gigabit per second between the two. I'll try with version 3.1.3.

**Comment by sahlst** [comment]
Hi, I'm also having this issue, real anticlimax. :(
Client version is 3.2.1 and server 21.0.1.
This is my first time setting up Nextcloud and I am really new to this but let me know if I can help with providing info in any way, logs et.c.? I really would like to see this thing work so I can leave Dropbox. :)

**Comment by Temtaime** [comment]
Just updated to 3.2.1 from 3.0.1.
It started to sync my whole C: drive. Very disgusting. Removed appdata/nextcloud folder and set up from stratch.
Now it says "error writing metadata to the database" and doesn't sync anything.

**Comment by freeone3000** [comment]
3.1.3 works. This appears to be a 3.2.1 regression.

**Comment by sahlst** [comment]
I uninstalled 3.2.1 and installed 3.1.3. The application seems to start synching but it crashes after some 10-30 seconds. The app first shows green check icon in the system tray but actually says "Starting synchronisation" (instead of the message in 3.2.1 which shows "waiting") but it crashes almost immediately. :(

BTW my Win 10 version is 19042.985 which I believe is the latest. I could upload some logs if I only knew how to go about that

**Comment by kagstrom2100** [comment]
> I uninstalled 3.2.1 and installed 3.1.3. The application seems to start synching but it crashes after some 10-30 seconds. The app first shows green check icon in the system tray but actually says "Starting synchronisation" (instead of the message in 3.2.1 which shows "waiting") but it crashes almost immediately. :(
> ![Skärmbild 2021-05-21 004647](https://user-images.githubusercontent.com/84497755/119058110-08fafc80-b9ce-11eb-85b1-ce5037c1c6e0.png)
> 
> BTW my Win 10 version is 19042.985 which I believe is the latest. I could upload some logs if I only knew how to go about that.
> 
> Also interestingly, as seen in the screenshot the client actually seems to recognise the files on the server, also the 3.1.3 version does state # of files et.c. while the 3.2.1 v would only say "waiting" for synchronisation. (Sorry I'm unable to translate 100% since I run the Swedish language version of the client :S)

Got this exakt same thing on my end, also running the latest Windows update!

**Comment by FlexW** [comment]
@kagstrom2100 @sahlst To prevent the crash when downgrading, you need to delete the sync journal (downgrading is not officially supported). To do so, delete the files starting with `._sync*` in the sync folder.

**Comment by kagstrom2100** [comment]
@FlexW Thanks! That made it work once, after I removed all the files (like a few mb of files) it went back to the "Synchronizing with local folder" status, but not actually performing any sync at all.

**Comment by kagstrom2100** [comment]
So I just tried to downgrade to 3.0.3 and it seems to work much better, so think I'll stick with that until this issue is fixed.

**Comment by Meth0d007** [comment]
Same here on all my W10 Clients v3.2.2. 
```

2021-06-23 20:12:29:716 [ info nextcloud.sync.networkjob.lscol ]:	LSCOL of QUrl("XXXX/S6-Bilder/") FINISHED WITH STATUS "OK"
2021-06-23 20:12:29:716 [ debug nextcloud.sync.networkjob ]	[ OCC::AbstractNetworkJob::slotFinished ]:	Network job OCC::LsColJob finished for "/S6-Bilder/"
```

But it never does anything in those folders.

![image](https://user-images.githubusercontent.com/17513721/123148095-1e959300-d460-11eb-9aca-27175097ddd2.png)

**Comment by nathanielhudson** [comment]
I want to add that I'm seeing (I believe) the same issue on 3.2.3 and 3.2.4.

Attempting to download a large number of files crashes or hangs nextcloud, and it continues to hang on every subsequent launch. Syncing the "proxy" versions of these files works fine, but when I tell nextcloud to keep them on my device the software hangs and no files are downloaded.

Downgrading to 3.1.3 resolves my issue.

**Comment by Meth0d007** [comment]
Yes same here. I got it somehwat working on a client by deleting all ._sync_ files an resyncing every client that has this account. But im not sure if this sticks, i have lost all trust in this sync so i am double checking everything, very frustrating.

**Comment by freeone3000** [comment]
Seems to be fixed as of 3.2.4
